pages/thc/[processRef]/pause.tsx

```diff
@@ -8,23 +8,41 @@ import { NextPage } from "next";
 import { useRouter } from "next/router";
 import React, { useState } from "react";
 import ProgressBar from "../../../components/ProgressBar";
+import getProcessRef from "../../../helpers/getProcessRef";
 import persistProcessData from "../../../helpers/persistProcessData";
 import urlsForRouter from "../../../helpers/urlsForRouter";
+import useDataValue from "../../../helpers/useDataValue";
 import useOnlineWithRetry from "../../../helpers/useOnlineWithRetry";
 import MainLayout from "../../../layouts/MainLayout";
 import PageSlugs, { urlObjectForSlug } from "../../../steps/PageSlugs";
 import PageTitles from "../../../steps/PageTitles";
+import Storage from "../../../storage/Storage";
 
 const PausePage: NextPage = () => {
   const router = useRouter();
+  const processRef = getProcessRef(router);
   const online = useOnlineWithRetry();
   const [progress, setProgress] = useState(0);
   const [saving, setSaving] = useState(false);
   const [saveError, setSaveError] = useState();
 
-  // TODO: Replace these with the real values!
-  const address = "1 Mare Street, London, E8 3AA";
-  const tenants = ["Jane Doe", "John Doe"];
+  const tenants = useDataValue(
+    Storage.ExternalContext,
+    "residents",
+    processRef,
+    (values) => (processRef ? values[processRef]?.tenants : undefined)
+  );
+
+  const tenantLab = tenants.result?.map((label) => label.fullName).join(", ");
```

Comment:
```suggestion
  const tenantLabel = tenants.result?.map((label) => label.fullName).join(", ");
```

Let's use the full word.